### DISCUSSION TITLE
A way to access the name of a request

### DISCUSSION THREAD

**Discussion by stray-signals** [usebruno/bruno#3195] [General]
My team is trying to make some dynamic standardized testing solution, which has us actually writing the tests as JavaScript files that a request pulls in and runs. We want to set up an index to make that process a bit easier and were wondering if there is anyway to get the requests name? Like I see the name field in the meta section of the .bru file but is there anyway to access that information from the test section?

**Comment by That-1-QA** [comment]
I wish there was something like this. I have some  tests that need to re-issue the same request multiple times, polling for the status of a job, until the status is complete.

In postman I could accomplish this with this within my tests `postman.setNextRequest(request.name);`  
With bruno. I haven't found a way to do this, and have just been copying the request name and hard coding it within the tests...

**Reply by tlaloc911** [reply]
If #3298 gets merged, you could get request's full path name

**Comment by helloanoop** [comment]
This is being solutioned in #3817

Please track #3817 for updates and to shared any feedback.
